hr_utilization_report/models/hr_employee.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    @api.multi
+    def action_utilization_report_wizard(self):
+        return {
```

Comment:
The same for this

Reply:
I'm not against putting code in XML, though I think code should be code and markup should be markup.